packages/components/tree-v2/src/emits.ts

```diff
@@ -0,0 +1,17 @@
+export const NODE_CLICK = 'node-click'
```

Comment:
Also, in order to better support typescript, please refer to `packages/components/affix/src/affix.ts` to define types for emits.